Makefile

```diff
@@ -0,0 +1,20 @@
+OJS := $(wildcard *.js)
+MJS := $(OJS:.js=.min.js)
+DJS := $(addprefix dist/,${MJS})
+
+all: $(DJS)
+.PHONY: all
+
+dist/%.min.js : %.js
```

Comment:
Would you mind explaining these two targets please?

Reply:
which 2 ?